bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -0,0 +1,263 @@
+        </p>
+
+        <p>
+          {{ _('Send Tabs easily lets you share open tabs between mobile and desktop.') }}
+        </p>
+      </section>
+
+      <section id="privacy">
+        <h3>{{ _('Privacy') }}</h3>
+
+        <em>{{ _('iOS and Android') }}</em>
+
+        <p>
+          {{ _('Control what you share online with the most powerful Private Browsing mode with added Tracking Protection.') }}
+        </p>
+
+        <p>
+          {{ _('Stops ads that track and follow you around the Internet.') }}
+        </p>
+
+        <p>
+          {{ _('Won’t remember your browsing history or save cookies after you’re done.') }}
+        </p>
+      </section>
+
+      <section id="extensions">
+        <h3>{{ _('Extensions & Customization') }}</h3>
+
+        <em>{{ _('Android only') }}</em>
+
+        <p>
+          {{ _('Break out of the browser box. From emojis to productivity tools, personalize Firefox to suit your style.') }}
+        </p>
+
+        <h4>{{ _('Extensions') }}</h4>
+        <p>
+          {{ _('Choose from thousands of tools like ad blockers, video downloaders and password managers.') }}
+        </p>
+
+        <h4>Themes</h4>
+        <p>
+          {{ _('Match Firefox to your mood. Get a new look from our theme categories or create your own.') }}
+        </p>
+      </section>
+    </article>
+
+    <article id="focus">
+      <h2>{{ _('Firefox Focus') }}</h2>
+
+      <p>
+        {{ _('Blocks ads with trackers. Automatically. Plus, it can even speed up page loading. Get the private browser that has your back.') }}
+      </p>
+
+      <nav class="page-nav" id="focus-page-nav">
+        <strong>{{ _('Firefox Focus') }}</strong>
+
+        <button type="button" class="button quantum">{{ _('Download now') }}</button>
+        <ol>
+          <li><a href="">{{ _('Automatic Privacy') }}</a></li>
+          <li><a href="">{{ _('Speed') }}</a></li>
+          <li><a href="">{{ _('Lightweight') }}</a></li>
+        </ol>
+
+        <a href="">{{ _('Previous') }}</a>
+        <a href="">{{ _('Next') }}</a>
+      </nav>
+
+      <section id="automatic-privacy">
+        <h3>{{ _('Automatic Privacy') }}</h3>
+
+        <em>{{ _('iOS and Android') }}</em>
+
+        <p>
+          {{ _('Browse stress free. Powerful privacy features are always on and your browsing history is wiped after every session.') }}
+        </p>
+
+        <p>
+          {{ _('Automatically blocks ads that have sneaky hidden trackers that collect data.') }}
+        </p>
+
+        <p>
+          {{ _('One tap erases your cookies and browsing history.') }}
+        </p>
+      </section>
+
+      <section id="speed">
+        <h3>{{ _('Speed') }}</h3>
+
+        <p>
+          {{ _('Get more done. By blocking some ads and scripts, Tracking Protection can speed up your browsing.') }}
+        </p>
+
+        <p>
+          {# L10n: The sentence below will likely change soon to include a percentage, e.g. "...can mean 44% faster loading pages". #}
```

Comment:
Looks like double quotes in comments don't parse well. What does "soon" mean?

Reply:
I believe as soon as they find a source verifying a percentage, but I don't know what priority this has. Page stakeholders know they missed the deadline for copy and will have to file a bug for the change. We can ship without it.

FWIW, the original copy included "44%", but was changed due (I think) to lack of verification.